docs/how_to/documenting_new_feature_toggles.rst

```diff
@@ -77,18 +87,98 @@ Temporary (``.. toggle_use_cases: temporary``):
 * Use Case 7: Opt-out/Opt-in (expected timeline: 2 years, ``.. toggle_use_cases: opt_out`` or ``.. toggle_use_cases: opt_in``)
 * Use Case 8: Open edX option (expected timeline: 3 years, ``.. toggle_use_cases: open_edx``)
 
-Note that if ``toggle_use_cases`` is "temporary" then ``toggle_target_removal_date`` **must** be defined.
+Note that if ``toggle_use_cases`` includes "temporary" then ``toggle_target_removal_date`` **must** be defined.
 
-Additional resources
-====================
-
-For more details on the individual annotations, see `OEP-17 <https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html#specification>`__.
+Additional details
+==================
 
-The documentation format used to annotate feature toggles is stored in the code-annotations repository: `feature_toggle_annotations.yaml <https://github.com/edx/code-annotations/blob/master/code_annotations/contrib/config/feature_toggle_annotations.yaml>`__.
+Multi-line annotations
+----------------------
 
-Note that all annotation fields can be wrapped on multiple lines, as long as every line after the first is prefixed by at least two empty spaces, plus the base indentation of the first line. For instance::
+Note that all annotation fields can be wrapped on multiple lines, as long as every line after the first is prefixed by at least three empty spaces, two spaces plus the base indentation of the first line. For instance::
 
     # .. toggle_description: line 1
     #   line2
 
+Plain-text formatting
+---------------------
+
 Note also that the annotation values will be considered as raw text and will not be parsed in any way. For instance, links and text formatting such as bold, italic or verbatim tags are currently unsupported. This might change in the future.
+
+Long-lines
+----------
+
+If you have a really long line, for example with a url that you don't want to break up, you may need to disable pylint using the following::
+
+    # .. toggle_tickets: https://some.com/long/url/that/you/dont/want/to/break/up.rst  # pylint: disable=line-too-long,useless-suppression
```

Comment:
Why the "useless-suppression"?

Reply:
It worked? I didn't have time to look into it. During the docathon, it came up a couple of times and I didn't look into it.  You are welcome to improve. :)  Not sure if this is true in all repos, or just edx-platform?

Reply:
I'd like to get a thumb and merge the PR, and we can iterate on this line/section if we wish.  I have other changes that may go in, so it would be good to land this base.